casper/src/test/rholang/FailingResultCollectorTest.rho

```diff
@@ -1,8 +1,16 @@
 //scalapackage coop.rchain.rholang
 
-new assert(`rho:test:assert`), assertEquals(`rho:test:assertEquals`)  in {
+new assert(`rho:test:assertAck`),
+    testCompleted(`rho:test:testSuiteCompleted`),
+    ack
+in {
+  assert!("false should fail", false, "clue assertTrue", *ack) |
 
-  assert!("false should fail", false) |
+  for (_ <- ack) {
+    assert!("0 != 1 should fail", (0, "==", 1), "clue assertEquals", *ack) |
```

Comment:
```suggestion
    assert!("0 == 1 should fail", (0, "==", 1), "clue assertEquals", *ack) |
```

Reply:
this was rebased on top of PR 2100 because I needed the tests. Please ignore the rest of the changes except MakeMint.rho and MakeMintTest.rho